.github/workflows/ci-build.yaml

```diff
@@ -261,11 +262,14 @@ jobs:
         run: docker load < /tmp/argoexec_image.tar
       - name: Set-up /etc/hosts
         run: |
-          echo '127.0.0.1 dex'      | sudo tee -a /etc/hosts
-          echo '127.0.0.1 minio'    | sudo tee -a /etc/hosts
-          echo '127.0.0.1 postgres' | sudo tee -a /etc/hosts
-          echo '127.0.0.1 mysql'    | sudo tee -a /etc/hosts
-          echo '127.0.0.1 azurite'  | sudo tee -a /etc/hosts
+          echo '127.0.0.1 dex'               | sudo tee -a /etc/hosts
```

Comment:
These changes haven't been applied to [devcontainer](https://github.com/argoproj/argo-workflows/blob/main/.devcontainer/devcontainer.json) nor explained in [the developer documentation](https://github.com/argoproj/argo-workflows/blob/main/docs/running-locally.md).

It isn't clear that they can be applied there universally either.